BraintreePSP/BraintreePaymentScreensBuilder.swift

```diff
@@ -25,12 +25,18 @@ final public class BraintreePaymentScreenBuilder: PaymentScreenBuilder {
     public init(){}
 
     public func buildAddCardScreen(
+        guestMode: Bool,
```

Comment:
I would change the name of this parameter, to make it clear this value affects a payment method's saving behavior.